prod.yaml.jinja

```diff
@@ -88,19 +88,24 @@ services:
       - .docker/db-creation.env
     restart: unless-stopped
     {%- if postgres_exposed %}
+    {%- if traefik_version == 3 %}
     networks:
       default:
       inverseproxy_shared:
     labels:
       traefik.enable: "true"
       traefik.docker.network: "inverseproxy_shared"
-      {{- traefik2_labels.database(
+      {{- traefik3_labels_2.database(
         domains_prod,
         postgres_cidr_whitelist,
         _key,
         postgres_exposed_port,
         project_name,
       ) }}
+    {%- else %}
+    ports:
+      - "{{ postgres_exposed_port }}:5432"
```

Comment:
`postgres_exposed_port` should be a conditional question, only asked if using traefik 3 or newer.

Reply:
I added this because with Traefik 1 and 2, direct access to the database can be achieved by exposing the port and making some additional configuration in PostgreSQL. Although it's not too difficult to set the ports manually, I believe this should be set up to generate automatically if needed.

Reply:
Yes, I just mean that Copier shouldn't ask what port you want to expose if you're running on traefik 3 and thus you're not gonna expose it.

Reply:
You're right, I didn't understand you correctly. Thanks!